app/screens/Main.tsx

```diff
@@ -1,13 +1,16 @@
 import { StatusBar } from 'expo-status-bar'
 import React from 'react'
 import { SafeAreaProvider } from 'react-native-safe-area-context'
+import ErrorBoundary from './ErrorBoundary/ErrorBoundary'
 import { RootNavigator } from './RootNavigator'
 
 export function Main (): JSX.Element {
   return (
-    <SafeAreaProvider>
-      <RootNavigator />
-      <StatusBar />
-    </SafeAreaProvider>
+    <ErrorBoundary>
```

Comment:
Should move ErrorBoundary to `App.tsx` at top level since errors can occur there.

Reply:
updated